package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vso-android-extension",
-  "version": "2.164.1",
+  "version": "2.179.1",
```

Comment:
Could you please fix the version to `2.179.0`?

Also, @anatolybolshakov please correct me if I wrong, but since this release contains changes only for documentation we don't have to bump up the version of the tasks.

Reply:
Agreed, there's no need to bump task version - since we update only readme file.